assignments/arrays.js

```diff
@@ -63,32 +63,54 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let car33 = 0
 
+console.log(`Car 33 is a ${inventory[32].2011} ${inventory[32].Jeep} ${inventory[32].Wrangler}` );
```

Comment:
Make sure you are referencing the `key` and not the value in the object. Instead of doing a dot notation for `Wrangler`, you should do it for `car_model` instead.